docs/docs-content/clusters/edge/site-deployment/site-installation/initial-setup.md

```diff
@@ -36,15 +36,32 @@ perform the setup, you can issue the command `palette-tui` in the terminal to tr
 
-   If you are accessing the Edge host with an SSH connection, you can issue the command `palette-tui` to bring up the
-   TUI. You can also use the same command to bring up the TUI if you have gone through the initial setup in the TUI and
-   want to change any configuration. However, you can only do this before you have deployed a cluster on the Edge host.
+   :::warning
 
-2. If you have already configured a user in your **user-data** file in the EdgeForge step, this step will be skipped
-   automatically.
+   The following table describes each option in the GRUB screen for your information. However, you should never manually
+   change the boot option. Doing so may cause Edge cluster deployment failures.
+
+   :::
+
+   | Option                                          | Description                                                                                                                                                                                                                                                           |
+   | ----------------------------------------------- | --------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+   | Palette eXtended Kubernetes - Edge              | This option boots the active image on your Edge host. If the Edge host is already part of a cluster, the active image is the image that your cluster uses.                                                                                                            |
+   | Palette eXtended Kubernetes - Edge (fallback)   | This option boots the passive image on your Edge host. If you have updated the cluster with a new cluster profile, the passive image is the image used by the previous version of the cluster.                                                                        |
+   | Palette eXtended Kubernetes - Edge recovery     | This option boots the recovery image. The recovery image is the OS image in the installer ISO. It does not include Kubernetes. The Palette agent is also inactive when this option is used, which means that the Edge host will not attempt to register with Palette. |
+   | Palette eXtended Kubernetes - Edge Reset        | This option wipes all data and resets the Edge host to the state after a fresh installation. The difference between this option and the recovery option is that the reset option wipes all application data and the Palette agent will be active after a reset.       |
+   | Palette eXtended Kubernetes - Edge Registration | If the Edge host is already part of a cluster, this option will not appear. This option boots the Edge host to the registration phase. For more information, refer to [Edge Host Registration](../../site-deployment/site-installation/edge-host-registration.md).    |
```

Comment:
The registration option will still appear if the edge host is part of the cluster.

Reply:
ummm this is not the case for uki at least @kpiyush17

Reply:
I have verified that the registration indeed still appears even if the Edge host is part of a cluster in non-UKI situations. I think i am just going to delete that first sentence.